internal/pkg/service/buffer/sink/tablesink/storage/volume/collection_test.go

```diff
@@ -1,61 +1,92 @@
-package volume
+package volume_test
 
```

Comment:
Switched to a blackbox test to prevent circular references `volume -> test -> volume.Volume`.

Another solution is to copy the test structure test.Volume to this pkg.
​